fundamentals/android/auth-sign-in-with-google/Notes/app/src/main/java/com/notes/app/screens/authentication/sign_in/SignInViewModel.kt

```diff
@@ -56,7 +56,7 @@ class SignInViewModel @Inject constructor(
         }
     }
 
-    fun onSignUpClick(openAndPopUp: (String, String) -> Unit) {
-        openAndPopUp(SIGN_UP_SCREEN, SIGN_IN_SCREEN)
+    fun onSignUpClick(openScreen: (String) -> Unit) {
+        openScreen(SIGN_UP_SCREEN)
     }
```

Comment:
(not blocking, this can be done in a follow-up PR)

Does this action need to be performed in the ViewModel? Seems like it's just navigating to different screen, which probably could be done in the UI layer.

Reply:
You're right, it could be performed in the UI layer - but this is a larger task as all other ViewModels are following this pattern. I'll do that in a separate PR.